### ISSUE TITLE
NCHG error

### ISSUE THREAD

**Issue by sebastian-gregoricchio** [Chrom3D/pipeline#1]
Dear all,
I am trying to generate the GTrack from my HiC data. 

When I run this command: 

```
NCHG \
-m 50000 \
-p ${INTRA_CHR_DIR}/${SAMPLE}_50kb.domain.RAW.no_cen.bedpe \
> ${SAMPLE}_50kb.domain.RAW.no_cen.NCHG.out
```

I get the following error message:
```
NCHG: CCCsig/CCCMatrix.cpp:125: void CCCMatrix<T>::setElement(Segment, Segment, T) [with T = int]: Assertion `this->isValidRow(row)' failed.
Aborted (core dumped)
```

Do you know what does it mean or to what it could be due to?

Thanks in advance for your help!

**Comment by Simple53** [comment]
same error. Have you figured it out? @sebastian-gregoricchio

**Comment by sebastian-gregoricchio** [comment]
> same error. Have you figured it out? @sebastian-gregoricchio 

Unfortunately not

**Comment by jonnings** [comment]
Could you maybe share the .bedpe file that causes this issue?

**Comment by Simple53** [comment]
[full_genome_bead_interactions.intra.nocen.bedpe.txt](https://github.com/Chrom3D/pipeline/files/12279239/full_genome_bead_interactions.intra.nocen.bedpe.txt)
Here is the .bedpe file

**Comment by jonnings** [comment]
The problem seems to be caused by the fact that you have non-integer values for the contact frequencies in the 7th column.  You should use non-normalized counts for the statistical method to make sense. Let me know if this helps.

**Comment by Simple53** [comment]
Yeah, it was in fact a problem of using normalized counts. I have solved it by using raw counts. Thanks!! @jonnings
